sdk/openai/Azure.AI.OpenAI/tests/Samples/SpeechGeneration.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Identity;
+using NUnit.Framework;
+
+namespace Azure.AI.OpenAI.Tests.Samples;
+
+public partial class SpeechGeneration
+{
+    [Test]
+    [Ignore("Only verifying that the sample builds")]
+    public async Task TranscribeAudio()
+    {
+        string endpoint = "https://myaccount.openai.azure.com/";
+        OpenAIClient client = new(new Uri(endpoint), new DefaultAzureCredential());
+
+        #region Snippet:SpeechGeneration
+        string text = "Hello World";
+        AudioSpeechOptions speechOptions = new()
+        {
+            Input = text,
+            DeploymentName = "my-tts-deployment", // tts-1 as model name for non-Azure OpenAI
+            Voice = AudioSpeechVoice.Alloy,
+            ResponseFormat = AudioSpeechOutputFormat.Mp3,
+            Speed = 0.8f
+        };
+
+        Response<BinaryData> response = await client.GetAudioSpeechAsync(speechOptions);
+
+        File.WriteAllBytes("myAudioFile.mp3", response.Value.ToArray());
+        #endregion
+
```

Comment:
Please remove the blank line.

Reply:
Resolve by https://github.com/ninghu/azure-sdk-for-net/pull/11/commits/de73afce474165f9b31c22fb85dee73ec798bb03